tests/unit/test_all_paths.cpp

```diff
@@ -307,3 +307,67 @@ TEST_F(AllPathsTest, DestinationSpecificPaths) {
 	Graph_Free(g);
 }
 
+
+// Test all paths of the minimum length from source to a specific destination node.
+TEST_F(AllPathsTest, OnlyShortestPaths) {
```

Comment:
this test is a bit unclear to me, is it trying to find all shortest paths between a node (`src`) and itself ?

Reply:
That's correct, yes.